core/docker-entrypoint.sh

```diff
@@ -56,8 +63,10 @@ if [ "$(id -u)" = '0' ] && [ "${QUESTDB_DATA_DIR%/}" != "/root/.questdb" ] && [
     if [ "$DO_CHOWN" = "true" ]; then
         find_and_own_dir $QUESTDB_UID $QUESTDB_GID
     fi
+    echo "Executing: gosu $QUESTDB_UID:$QUESTDB_GID $@"
```

Comment:
nit: I don't know if we should print the full command - some users might have some sensitive arguments there. Same consideration for L71 (`echo "Executing: $@"`). WDYT?

Reply:
Gotcha, makes sense, will remove